src/App.js

```diff
@@ -30,7 +30,7 @@ const App = () => (
                 <Link to="/quotes">Quote</Link>
               </li>
             </ul>
-          </div>
+          </nav>
         </div>
       </header>
```

Comment:
Since the first CR marked the requirement as an OPTIONAL change, even though it should have been a mandatory one, it's okay for the second CR to bring it up again as OPTIONAL.

Reply:
Hi everyone, I made this comment as an OPTIONAL change because it is a requirement for a previous milestone, not for this one. But I am giving the opportunity for the student to improve his repo with an OPTIONAL change request. :pray: